src/components/combo/combo.ts

```diff
@@ -325,6 +325,18 @@ export default class IgcComboComponent<T extends object>
     return html`${this.groupKey && item[this.groupKey]}`;
   };
 
+  /**
+   * Sets the component's positioning strategy.
+   */
+  @property({ attribute: 'position-strategy' })
+  public positionStrategy: 'absolute' | 'fixed' = 'fixed';
```

Comment:
This doesn't seem to work or do much at least. This bit I assume is important:
> Ensure your initial layout matches the strategy:
#tooltip { position: fixed; ....
https://floating-ui.com/docs/computePosition#strategy

So here:
https://github.com/IgniteUI/igniteui-webcomponents/blob/94c199bd5c91a048cd1ad098f56a0b2bddf6a489/src/components/combo/combo.ts#L867-L872
You need something like the dropdown does:
https://github.com/IgniteUI/igniteui-webcomponents/blob/94c199bd5c91a048cd1ad098f56a0b2bddf6a489/src/components/dropdown/dropdown.ts#L523-L525
![image](https://github.com/IgniteUI/igniteui-webcomponents/assets/3198469/0a98e699-c197-4de2-8490-108b904aa464)